src/test/kotlin/dev/stalla/writer/namespace/GoogleplayWriterTest.kt

```diff
@@ -72,7 +72,7 @@ internal class GoogleplayWriterTest : NamespaceWriterTest() {
         val podcast = aPodcast(
             googleplay = aPodcastGoogleplay(
                 author = " ",
-                owner = " ",
+                email = " ",
```

Comment:
I'm still baffled as to why the docs are so wrong compared to the dtd

Reply:
Yep. Every time I have a look I notice something else. At this point I'm just afraid to open the XSD again and never getting the v1.0 out 😅

Reply:
How I understand! ahaha